package.json

```
@@ -40,7 +40,7 @@
   "author": {
     "name": "James Halliday",
     "email": "mail@substack.net",
-    "url": "http://substack.net"
+    "url": "http://browserify.org/"
```

the author URL can stay at substack.net (since it's a personal field and not one for the project)
```suggestion
    "url": "http://substack.net"
```